### PR TITLE
EWPP-2559: ECL table style cleanup.

### PR DESCRIPTION
**Pull request by sergepavle**
### Description

[Insert description here]

### Change log

- Added:
- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

File: modules/oe_theme_helper/src/Plugin/Filter/FilterEclTable.php [hunk 1]
```diff
@@ -37,12 +37,40 @@ public function process($text, $langcode) {
...
     $xpath = new \DOMXPath($dom);
 
-    foreach ($xpath->query('//table[.//th]') as $table) {
-      // Skip the table if any cell spans over multiple columns or rows.
-      $span_cells = $xpath->query('.//*[self::th or self::td][(@colspan and @colspan > 1) or (@rowspan and @rowspan > 1)]', $table);
-      if ($span_cells->count() !== 0) {
+    foreach ($xpath->query('//table') as $table) {
+      // Put ECL related classes for table tag.
+      $this->elementAddClass($table, 'ecl-table');
+      // Add related to "Zebra striping" classes.
```
Comment (nagyad) [inline modules/oe_theme_helper/src/Plugin/Filter/FilterEclTable.php]: This sentence is not correct in English.

File: modules/oe_theme_helper/src/Plugin/Filter/FilterEclTable.php [hunk 2]
```diff
@@ -57,42 +85,59 @@ public function process($text, $langcode) {
       $has_header_row = $xpath->query('(./thead/tr[1])[count(./*[not(self::th)]) = 0]', $table);
       if ($has_header_row->count()) {
         $header_row = $has_header_row[0];
-        foreach ($xpath->query('./th', $header_row) as $cell) {
-          $headers[] = $cell->nodeValue;
+        foreach ($xpath->query('./th', $header_row) as $thead_cell) {
+          $this->elementAddClass($thead_cell, 'ecl-table__header');
+          // Add related to "Sort" data attribute.
```
Comment (nagyad) [inline modules/oe_theme_helper/src/Plugin/Filter/FilterEclTable.php]: This sentence is not correct in English.

**Review by nagyad — CHANGES_REQUESTED** [review]

File: modules/oe_theme_helper/src/Plugin/Filter/FilterEclTable.php (hunk 1 above)
Comment (sergepavle) [reply modules/oe_theme_helper/src/Plugin/Filter/FilterEclTable.php]: Fixed.

File: modules/oe_theme_helper/src/Plugin/Filter/FilterEclTable.php (hunk 2 above)
Comment (sergepavle) [reply modules/oe_theme_helper/src/Plugin/Filter/FilterEclTable.php]: Fixed.

**Review by nagyad — DISMISSED** [review]

**Review by yenyasinn — DISMISSED** [review]

**Review by nagyad — APPROVED** [review]

**Review by yenyasinn — APPROVED** [review]

**Pull request merged**